sdk/src/main/java/com/karhoo/sdk/api/network/client/APITemplate.kt

```diff
@@ -235,6 +246,25 @@ interface   APITemplate {
     @GET(LOYALTY_CONVERSION)
     fun loyaltyConversionRates(@Path(IDENTIFIER_ID) id: String): Deferred<Resource<LoyaltyConversion>>
 
+    @GET(LOYALTY_STATUS)
+    fun loyaltyStatus(@Path(IDENTIFIER_ID) id : String): Deferred<Resource<LoyaltyStatus>>
+
+    @GET(LOYALTY_BURNPOINTS)
+    fun loyaltyBurnPoints(@Path(IDENTIFIER_ID) id: String, @Path(IDENTIFIER_CURRENCY) currency:
+    String, @Query(IDENTIFIER_AMOUNT) amount: Int): Deferred<Resource<LoyaltyPoints>>
+
+    @GET(LOYALTY_EARNPOINTS)
+    fun loyaltyPointsToEarn(@Path(IDENTIFIER_ID) id: String, @Path(IDENTIFIER_CURRENCY) currency:
+    String, @Query(IDENTIFIER_TOTALAMOUNT) totalAmount: Int, @Query(IDENTIFIER_BURNPOINTS)
```

Comment:
The identation seems broken here, we should have params on separate lines for easier reading. Same applies to above